codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/CargoTomlGenerator.kt

```diff
@@ -27,15 +27,16 @@ class CargoTomlGenerator(
             "package" to mapOf(
                 "name" to settings.moduleName,
                 "version" to settings.moduleVersion,
-                "description" to settings.moduleDescription,
                 "authors" to settings.moduleAuthors,
-                "license" to settings.license,
+                "description" to settings.moduleDescription,
```

Comment:
I wonder if we should add moduleDescription validation...

Reply:
What kind of validation are you thinking of? That a description is provided, or that it is short and doesn't contain HTML?